core/server/common/src/main/java/alluxio/RpcUtils.java

```diff
@@ -106,6 +108,7 @@ private RpcUtils() {} // prevent instantiation
   public static <T> T callAndReturn(Logger logger, RpcCallableThrowsIOException<T> callable,
       String methodName, boolean failureOk, String description, Object... args)
       throws StatusException {
+    Metrics.MASTER_RPC_OPS_THROUGHPUT.mark(1);
     // avoid string format for better performance if debug is off
     String debugDesc = logger.isDebugEnabled() ? String.format(description, args) : null;
     try (Timer.Context ctx = MetricsSystem.timer(getQualifiedMetricName(methodName)).time()) {
```

Comment:
Could we add a new reload method `timer` can accept more than one `MetricsName`? If so, it would reduce the metric related code in the main logic.

One possible usage like

```java
try (Timer.Context ctx = MetricsSystem.timer(getQualifiedMetricName(methodName), "rpcCall").time()) {
...
}
```

If we want to count the total `InProgress` call, we may want to use 

```java
MetricsSystem.counter(getQualifiedInProgressMetricName(methodName), "rpcCallInProgress").inc();
```

Reply:
I updated the PR similar to what you proposed.
PTAL